Comparing source compatibility of /root/workspace/bowenli86/flink/389031126/flink-streaming-java/target/flink-streaming-java_2.11-1.6-SNAPSHOT.jar against /root/./workspace/bowenli86/flink/389031126/.m2/org/apache/flink/flink-streaming-java_2.11/1.5.0/flink-streaming-java_2.11-1.5.0.jar
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.streaming.api.environment.StreamExecutionEnvironment  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.DataStreamSource readFile(org.apache.flink.api.common.io.FileInputFormat, java.lang.String, int)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.DataStreamSource readTextFile(java.lang.String, int)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.DataStreamSource readTextFile(java.lang.String, java.lang.String, int)
